tcxparser.py

```diff
@@ -78,6 +78,18 @@ def altitude_avg(self):
         """Average altitude for the workout"""
         altitude_data = self.altitude_points()
         return sum(altitude_data)/len(altitude_data)
+    
+    @property
+    def altitude_max(self):
+        """Max altitude for the workout"""
+        altitude_data = self.altitude_points()
```

Comment:
Here's a question that I don't know the answer to... Is every TCX file guaranteed to have `ns:AltitudeMeters`? If not, or if the list can possibly be empty, then taking the `max` of an empty list will fail (and dividing by `len(altitude)` in `altitude_avg` will also fail).

Do you know what the spec says? It might be best to just assume that altitude_points can be empty, in which case we should check for that and return zero in these 2 methods and `altitude_avg`.